lib/qmp.ml

```diff
@@ -29,11 +29,20 @@ type enabled = {
   present: bool;
 }
 
+type vnc = {
+  enabled : bool;
+  auth    : string;
+  family  : string;
+  service : string;
```

Comment:
Thanks. Changed service to `int`.
For `family` and `host`, I don't think need to check them, they just returned by QEMU.